staging/src/k8s.io/api/networking/v1/types.go

```diff
@@ -141,10 +141,18 @@ type NetworkPolicyPort struct {
 	// +optional
 	Protocol *v1.Protocol `json:"protocol,omitempty" protobuf:"bytes,1,opt,name=protocol,casttype=k8s.io/api/core/v1.Protocol"`
 
-	// The port on the given protocol. This can either be a numerical or named port on
-	// a pod. If this field is not provided, this matches all port names and numbers.
+	// The port on the given protocol. This can either be a numerical or named
+	// port on a pod. If this field is not provided, this matches all port names and
+	// numbers.
+	// If present, only traffic on the specified protocol AND port will be matched.
 	// +optional
 	Port *intstr.IntOrString `json:"port,omitempty" protobuf:"bytes,2,opt,name=port"`
+
+	// EndPort, if set, indicates that a range of ports, from Port to EndPort inclusive
```

Comment:
```suggestion
	// endPort, if set, indicates that a range of ports, from Port to EndPort inclusive
```

in other files ^ is the pattern used.. not sure if the difference is intentional

Reply:
Yeah, I misread Andrew's comment about changing the name of the field vs the referenced in the description. I'll return the name of the variable in the files as upper-case but leave the description with the camelCase required by godoc

Reply:
Or maybe just leave as the Port description, that doesn't starts with "Port is blabla" but only "The port of [...]